test/module/irohad/model/command_validate_execute_test.cpp

```diff
@@ -212,6 +213,145 @@ TEST_F(AddAssetQuantityTest, InvalidWhenNoAsset) {
   ASSERT_FALSE(validateAndExecute());
 }
 
+class SubtractAssetQuantityTest : public CommandValidateExecuteTest {
+public:
+    void SetUp() override {
+      CommandValidateExecuteTest::SetUp();
+
+      asset = Asset();
+      asset.asset_id = asset_id;
+      asset.domain_id = domain_id;
+      asset.precision = 2;
+
+      wallet = AccountAsset();
+      wallet.asset_id = asset_id;
+      wallet.account_id = account_id;
+      wallet.balance = balance;
+
+      subtract_asset_quantity = std::make_shared<SubtractAssetQuantity>();
+      subtract_asset_quantity->account_id = creator.account_id;
+      Amount amount(100, 2);
+      subtract_asset_quantity->amount = amount;
+      subtract_asset_quantity->asset_id = asset_id;
+
+      command = subtract_asset_quantity;
+      role_permissions = {can_subtract_asset_qty};
+    }
+
+    decltype(AccountAsset().balance) balance = Amount(150ul, 2);
```

Comment:
What about auto?

Reply:
It's like https://github.com/hyperledger/iroha/blob/d143105918b961864e52a59c800c94c1db541898/test/module/irohad/model/command_validate_execute_test.cpp#L121

How about it?

Reply:
ping @kamilsa

Reply:
It's reason
```
~ iroha/test/module/irohad/model/command_validate_execute_test.cpp:241:5: error: 'auto' not allowed in non-static class member
    auto balance = Amount(150ul, 2);
    ^~~~
```